versions.tf

```diff
@@ -20,7 +20,17 @@ terraform {
     google = {
       source = "hashicorp/google"
       # TODO: Check why this filter was breaking tests.
-      # version = "~> 3.53, < 5.0"
+      version = ">= 3.53, < 5.0"
```

Comment:
Looks like this TPG version constraint is technically new, so we should probably mark this as a breaking change.